spec/requests/congress_messages_spec.rb

```diff
@@ -0,0 +1,185 @@
+require "rails_helper"
+
+RSpec.describe "Congress Messages", type: :request do
+  let!(:members) {
+    [FactoryGirl.create(:congress_member, state: "CA", bioguide_id: "C000880"),
+     FactoryGirl.create(:congress_member, state: "CA", bioguide_id: "A000360")]
+  }
+
+  let(:action_page) {
+    FactoryGirl.create(:action_page_with_congress_message)
+  }
+
+  let(:location) {
+    OpenStruct.new(success: true,
+                   street: "1630 Ravello Drive",
+                   city: "Sunnydale",
+                   zipcode: 94109,
+                   zip4: 1234,
+                   state: "CA",
+                   district: 10)
+  }
+
+  def stub_congress_forms_find_with_two_reps
+    stub_request(:post, /retrieve-form-elements/).
+      with(body: { "bio_ids" => ["C000880", "A000360"] }).
+      and_return(status: 200, body: file_fixture("retrieve-form-elements.json"))
+  end
+
+  def stub_congress_forms_find_with_one_rep
+    forms_body = JSON.parse(file_fixture("retrieve-form-elements.json").read)
+    forms_body.delete("A000360")
+    stub_request(:post, /retrieve-form-elements/).
+      with(body: { "bio_ids" => ["C000880"] }).
+      and_return(status: 200, body: forms_body.to_json)
+  end
+
+  before do
+    allow(SmartyStreets).to receive(:get_location).and_return(location)
+    stub_congress_forms_find_with_two_reps
+  end
+
+  describe "#new" do
+    subject do
```

Comment:
I think it would be more clear if the `subject` blocks were changed to methods with descriptive names